modules/express/test/unit/clientRoutes/externalSign.ts

```diff
@@ -102,6 +109,208 @@ describe('External signer', () => {
     envStub.restore();
   });
 
+  it('should read an encrypted prv from signerFileSystemPath and pass it to PaillierModulus, K, MuDelta, and S share generators', async () => {
```

Comment:
Any non-happy path tests we can add? What about invalid inputs/configs?

Reply:
there are a lot of missing non-happy path tests for the already supported eddsa external signer as well, i created this tickets to add non-happy path tests for external signer in general cause i think it would make this PR way too big. https://bitgoinc.atlassian.net/browse/HSM-120